site/src/pages/components/menu.explainer.mdx

```diff
@@ -367,6 +367,51 @@ There are ARIA guidelines for the [toolbar](https://www.w3.org/WAI/ARIA/apg/patt
 There is a history of attempting to do this in Webkit, for example for a [Touch Bar](https://github.com/whatwg/html/issues/3251). The idea for this is to collect all your "commands" and put them into the macOS menu bar (or expose them some other way). This proposal will not address this yet.
 
 
+## Reading List
+
+Menus have been heavily discussed, please see below to learn about the existing specification and efforts.
+
+### ARIA guidelines
+- the ARIA [menu role](https://w3c.github.io/aria/#menu)
+- the ARIA [menubar role](https://w3c.github.io/aria/#menubar)
+- the ARIA [menuitem role](https://w3c.github.io/aria/#menuitem)
+- the ARIA [menuitemcheckbox role](https://w3c.github.io/aria/#menuitemcheckbox)
+- the ARIA [menuitemradio role](https://w3c.github.io/aria/#menuitemradio)
+- the [Core AAM mappings](https://www.w3.org/TR/core-aam-1.2/#role-map-menu)
+- the [HTML AAM for <menu>](https://www.w3.org/TR/html-aam-1.0/#el-menu)
+
+### Relevant ARIA issues
+- [ARIA] [Consider allowing combobox to open menus](https://github.com/w3c/aria/issues/2050)
+
+### Relevant Navigation specific issues
+- [ARIA practices] [Clarify Purpose of Menu Navigation](https://github.com/w3c/aria-practices/issues/353)
+- [OpenUI] [Navigation vs menu items use case](https://github.com/openui/open-ui/issues/1193)
+- [ARIA] [Concerning navigation "menus" (open ui related)](https://github.com/w3c/aria/issues/2510)
+
+### APG pattern recommendations (note these are not guidelines)
+- [menu button](https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/)
+- [menubar](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/)
+- [menubar navigation example](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/examples/menubar-navigation/)
```

Comment:
adding the counter example to the menu navigation pattern - which achieves the UX of the menu navigation pattern, but doesn't use the menu/menuitem roles to do so

```suggestion
- [menubar navigation example](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/examples/menubar-navigation/)
- [disclosure navigation menu](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation/)
```